lib/folio_client/inventory.rb

```diff
@@ -22,5 +22,21 @@ def fetch_hrid(barcode:)
       result = client.get("/inventory/instances/#{instance_uuid}")
       result.dig("hrid")
     end
+
+    # @param hrid [String] folio instance HRID
+    # @param status_code [String] id for the "cat" instance status code
+    def cataloged?(hrid:, status_code:)
```

Comment:
Why is this called `cataloged?` instead of `status_code?`?

Reply:
I was probably thinking of this from the consumer perspective, but that makes less sense here, as you're pointing out. Will change this, which would also make it more generalizable.